pandas/tests/apply/test_series_apply.py

```diff
@@ -186,7 +186,7 @@ def test_apply_datetimetz():
     # change dtype
     # GH 14506 : Returned dtype changed from int32 to int64
```

Comment:
Seems like this comment is not fully true now since this was changed to int32?

Reply:
I've removed the comment, same below.